src/Cookie.php

```diff
@@ -111,29 +104,32 @@ public function setOptions(array $options): void {
         $this->options = $options;
     }
 
-    public function getCookie(string $name): mixed {
-        if (isset($_COOKIE[$name])) {
-            return $_COOKIE[$name];
+	/**
+	 * @return mixed|null
+	 */
+    public static function getCookie(string $name) {
+        if (!\array_key_exists($name, $_COOKIE)) {
```

Comment:
$_COOKIE is a global array. `isset` method can work but does many things what can bring unwanted side-effects (As example it will fail if cookie is defined but with `null` value). Since you want to test if it is there you can use the `array_key_exists` method.